site/_docs/howto.md

```diff
@@ -693,8 +693,11 @@ Before you start:
   is starting and therefore `master` branch is in code freeze until further notice.
 * Set up signing keys as described above.
 * Make sure you are using JDK 8 (not 9 or 10).
-* Make sure `master` branch and `site` branch are in sync, i.e. there is no commit on `site` that has not
-  been applied also to `master`. If you spot missing commits then port them to `master`.
+* Make sure `master` branch and `site` branch are in sync, i.e. there is no
+  commit on `site` that has not been applied also to `master`.
+  We are talking about the commit content, you need to pay attention to the commit message
+  and change, not hash: it is normal to have the same change in site and
```

Comment:
"site" and "master" are missing the backtick in the last sentence. Also please keep at the end the sentence: "If you spot missing commits then port them to `master`."

Reply:
Fixed. Thanks for your kind reminder.